dmarc.tf

```diff
@@ -1,8 +1,13 @@
+locals {
+  dmarc_policy = "v=DMARC1;p=reject;sp=reject"
+  dmarc_record = join(";", compact([local.dmarc_policy, var.dmarc_rua, var.dmarc_ruf]))
+}
+
```

Comment:
```suggestion
```

It felt cumbersome to repeat the `rua=` in the variable value (`dmarc_rua = "rua=email@address.com"`), so I changed it around a bit:

```hcl
resource "aws_route53_record" "dmarc" {
  count    = var.dmarc != null ? 1 : 0
  provider = aws.route53

  name    = "_dmarc.${var.domain}"
  ttl     = 600
  type    = "TXT"
  zone_id = data.aws_route53_zone.default.zone_id

  records = [join(";", compact([
    var.dmarc.policy,
    var.dmarc != null ? var.dmarc.rua != null ? "rua=${var.dmarc.rua}" : null : null,
    var.dmarc != null ? var.dmarc.ruf != null ? "ruf=${var.dmarc.ruf}" : null : null,
  ]))]
}
```

Usage now looks something like this:

```
module "ses" {
  providers = { aws = aws, aws.route53 = aws }
  source    = "../.."
  domain    = "mydomain.com"
  tags      = {}

  dmarc = {
    rua = "mailto:email@address.com"
  }
}

```